src/main/java/com/czertainly/core/dao/entity/cmp/CmpTransaction.java

```diff
@@ -0,0 +1,64 @@
+package com.czertainly.core.dao.entity.cmp;
+
+import com.czertainly.core.dao.entity.Certificate;
+import com.czertainly.core.dao.entity.UniquelyIdentified;
+import com.fasterxml.jackson.annotation.JsonBackReference;
+import jakarta.persistence.*;
+
+import java.util.UUID;
+
+@Entity
+@Table(name = "cmp_transaction")
+public class CmpTransaction extends UniquelyIdentified {

```

Comment:
How do you know the state of the transaction? Is it needed? When is the transaction removed from the database? When there is no state or other indicators, can I send revocation request, or other requests, with the same transaction ID and different content multiple times?

Reply:
DONE, check if transaction exists (it stop processing; otherwise if operation is success/failed - transatction is stored)